packages/mockserver-config-writer/src/types/index.ts

```diff
@@ -0,0 +1 @@
+export * from './types';
```

Comment:
Do we really need that level of abstraction for a small component like this or could you just move all your types directly in to this file?

Reply:
With the change to the central type for `Editor` this file is gone now.